cmd/broadcaster-cli/app/keyset/balance/balance.go

```diff
@@ -14,20 +14,19 @@ import (
 var Cmd = &cobra.Command{
 	Use:   "balance",
 	Short: "Show balance of the keyset",
-	RunE: func(cmd *cobra.Command, args []string) error {
-
+	RunE: func(_ *cobra.Command, _ []string) error {
 		isTestnet, err := helper.GetBool("testnet")
 		if err != nil {
 			return err
 		}
-		wocApiKey, err := helper.GetString("wocAPIKey")
+		wocAPIKey, err := helper.GetString("wocAPIKey")
 		if err != nil {
 			return err
 		}
 
 		logger := helper.GetLogger()
 
-		wocClient := woc_client.New(!isTestnet, woc_client.WithAuth(wocApiKey), woc_client.WithLogger(logger))
+		wocClient := wocclient.New(!isTestnet, wocclient.WithAuth(wocAPIKey), wocclient.WithLogger(logger))
```

Comment:
I think `woc_client` could remain as it is, not?

Reply:
came back to `woc_client`